plugins/woocommerce-blocks/tests/e2e/tests/product-collection/product-collection.page.ts

```diff
@@ -798,6 +804,14 @@ class ProductCollectionPage {
 		}
 	}
 
+	async removeAdvancedPreview() {
+		await this.page.evaluate( () => {
+			if ( window.__removePreview ) {
+				window.__removePreview();
+			}
+		} );
+	}
+
```

Comment:
Let's not add this to the POM - it's tied to a very specific scenario, and we will likely not use it more than once.

Reply:
What do you mean with `POM`? You mean the `window` object?
Do we have a more general event handler that we could use? Or something like this: https://developer.mozilla.org/en-US/docs/Web/Events/Creating_and_triggering_events#the_old-fashioned_way ?

Reply:
Oooh yes, that makes sense :)
I originally was going to do that, but didn't have access to the `page` object, but maybe I didn't try hard enough to get access to that.

Reply:
Done: https://github.com/woocommerce/woocommerce/pull/51653/commits/0a7a5d84f1f23e9464b0bcda816bfad22326b486